src/Recurrence/Services/SubscriptionService.php

```diff
@@ -111,6 +111,7 @@ public function createSubscriptionAtMundipagg(PlatformOrderInterface $platformOr
     private function extractSubscriptionDataFromOrder(PaymentOrder $order)
     {
         $subscription = new Subscription();
+        $config = MPSetup::getModuleConfiguration();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Mundipagg\Core\Kernel\Abstractions\AbstractModuleCoreSetup' in method 'extractSubscriptionDataFromOrder'.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4895901)